### ISSUE TITLE
Error in if-expression: "if: ${{ !contains(github.event.head_commit.message, '#no-gha') }}" (Unable to dereference 'message' on non-struct 'invalid')

### ISSUE THREAD

**Issue by dodalovic** [nektos/act#1159]
The tool fails with the following message:

```shell
act -n
Error:   ❌  Error in if-expression: "if: ${{ !contains(github.event.head_commit.message, '#no-gha') }}" (Unable to dereference 'message' on non-struct 'invalid')
```

, when given such workflow...
 
```yaml
jobs:
  build-jar:
    runs-on: ubuntu-latest
    if: ${{ !contains(github.event.head_commit.message, '#no-gha') }}
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-java@v2
```

**Comment by adzimzf** [comment]
I face the same problem using the `act` latest version

**Comment by ChristopherHX** [comment]
Duplicate of #1071

The fix will be released in the next release v0.2.27.
However you can install the version based on latest commit read this section if you are using homebrew on linux/macOS https://github.com/nektos/act#homebrew-linuxmacos, otherwise build the master branch of this project from source e.g. `go build`.
